sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Processor/EventProcessorClientLiveTests.cs

```diff
@@ -8,10 +8,15 @@
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
+using Azure.Core;
```

Comment:
There are potentially some additional scenarios that we should include; I'm definitely interested in things that folks feel are meaningful and important.

Given time constraints, its unlikely that I'll include them in this round, but I will take notes and then open an issue to address them as a separate workstream.